src/java/voldemort/utils/MetadataVersionStoreUtils.java

```diff
@@ -144,6 +153,7 @@ private static String getPropertiesString(Properties props) {
     public static void setProperties(SystemStoreClient<String, String> versionStore,
```

Comment:
I left it on a fall back path, when the node by node set fails. Just to be cautious. We can gut it out after few months may be.